### DISCUSSION TITLE
Connecting to wifi image streamer

### DISCUSSION THREAD

**Discussion by scottjack0012** [bitcraze/discussions#665] [Q&A]
I have been trying to work through the AI deck examples and have gotten to the wifi streaming example : https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/
I've been having a lot of issues with this though:

1)I've found that I can only cfloader flash an example when using the 2022.05 firmware. So when I run $ cfloader flash cf2.bin stm32-fw -w [CRAZYFLIE_URI] the firmware is updated to the 2023 version and when I flash it doesn't get past 0%. Is it only meant to work with the 2022.05 firmware? Am I meant to ignore $ cfloader flash cf2.bin stm32-fw -w [CRAZYFLIE_URI]? Do i need the 2023 firmware to run the wifi streamer?
2) When going through the Getting started AI deck tutorial (https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/)
i was able to flash and run the .bin  wifi streamer example and the  ‘WiFi streaming example’ wifi would appear. Now I am getting to the one in the AI deck examples I can't find the wifi to connnect to after flashing the .img example. I wasn't sure what to change the SSID and KEY to in the menuconfig UI but I went back and set them to  WiFi streaming example and left the KEY blank. However there is still no wifi apprearing. Is there a specific name that I am meant to change the SSID to? Do I need to change any of the code to get it to work on the access point? I can still get the .bin flash to work and connect to the WiFi streaming example

I would be really grateful for any help 
Thanks
Jack

**Comment by knmcguire** [comment]
Hi!  I've just tested it out myself on crazyflie with the latest master of the crazyflie-firmware following the steps of the guide you gave. 
1. This is actually flashing the crazyflie firmware on the crazyflie with the kbuild config with the wifi settings, and I get the following:

```
cfloader flash cf2.bin stm32-fw -w radio://0/70/2M/E7E7E7E704
Reset to bootloader mode ...
Flashing 1 of 1 to stm32 (fw): 263359 bytes (258 pages) ..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10..........10........8
```

Then going to the instructions to flash the gap8 example of the latest development branch I get the following:

```
cfloader flash examples/other/wifi-img-streamer/BUILD/GAP8_V2/GCC_RISCV_FREERTOS/target.board.devices.flash.img deck-bcAI:gap8-fw -w radio://0/70/2M/E7E7E7E704
Reset to bootloader mode ...
Could not save cache, no writable directory
Could not save cache, no writable directory
Skipping bcAI:esp, not in the target list
Deck bcAI:gap8, reset to bootloader
| 0% Writing to bcAI:gap8 deck memory
/ 1% Writing to bcAI:gap8 deck memory
- 2% Writing to bcAI:gap8 deck memory
\ 3% Writing to bcAI:gap8 deck memory
...
- 98% Writing to bcAI:gap8 deck memory
\ 99% Writing to bcAI:gap8 deck memory
| 100% Writing to bcAI:gap8 deck memory

```
and I see the access point of the aideck popping up in my wifi list.

So there seems to not be a bug with the latest versions of the codes.

You shouldn't keep any of the keys blank I would say, so always give it an ssid and a password, as keeping those blank has not been tested.

What I would advise is is to start from the start: Start at the [getting started guide](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/) again with install the latest release 2023.02, just to make sure everything is all fresh. and then to go through the [wifi streamer guide](https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/wifi-streamer/) again
